diff --git a/node_modules/@vue/runtime-dom/dist/runtime-dom.d.ts b/node_modules/@vue/runtime-dom/dist/runtime-dom.d.ts
index 484fa67..58d12c8 100644
--- a/node_modules/@vue/runtime-dom/dist/runtime-dom.d.ts
+++ b/node_modules/@vue/runtime-dom/dist/runtime-dom.d.ts
@@ -1,4 +1,4 @@
-import { SetupContext, RenderFunction, ComputedOptions, MethodOptions, ComponentOptionsMixin, EmitsOptions, ComponentInjectOptions, SlotsType, ComponentOptionsWithoutProps, ComponentOptionsWithArrayProps, ComponentPropsOptions, ComponentOptionsWithObjectProps, ExtractPropTypes, DefineComponent, RootHydrateFunction, ConcreteComponent, BaseTransitionProps, FunctionalComponent, ObjectDirective, VNodeRef, RootRenderFunction, CreateAppFunction } from '@vue/runtime-core';
+import { SetupContext, RenderFunction, ComputedOptions, MethodOptions, ComponentOptionsMixin, EmitsOptions, ComponentInjectOptions, SlotsType, ComponentOptionsWithoutProps, ComponentOptionsWithArrayProps, ComponentPropsOptions, ComponentOptionsWithObjectProps, ExtractPropTypes, DefineComponent, RootHydrateFunction, ConcreteComponent, BaseTransitionProps, FunctionalComponent, ObjectDirective, VNodeRef, RootRenderFunction, CreateAppFunction, vShowOriginalDisplay, vShowHidden } from '@vue/runtime-core';
 export * from '@vue/runtime-core';
 import * as CSS from 'csstype';
 
@@ -119,7 +119,7 @@ export declare const withKeys: <T extends (event: KeyboardEvent) => any>(fn: T &
 
 declare const vShowOriginalDisplay: unique symbol;
 declare const vShowHidden: unique symbol;
-interface VShowElement extends HTMLElement {
+export interface VShowElement extends HTMLElement {
     [vShowOriginalDisplay]: string;
     [vShowHidden]: boolean;
 }
@@ -1310,4 +1310,5 @@ export declare const render: RootRenderFunction<Element | ShadowRoot>;
 export declare const hydrate: RootHydrateFunction;
 export declare const createApp: CreateAppFunction<Element>;
 export declare const createSSRApp: CreateAppFunction<Element>;
-
+export declare const vShowOriginalDisplay: typeof vShowOriginalDisplay;
+export declare const vShowHidden: typeof vShowHidden;
diff --git a/node_modules/@vue/runtime-dom/dist/runtime-dom.esm-bundler.js b/node_modules/@vue/runtime-dom/dist/runtime-dom.esm-bundler.js
index 387ccca..d39055a 100644
--- a/node_modules/@vue/runtime-dom/dist/runtime-dom.esm-bundler.js
+++ b/node_modules/@vue/runtime-dom/dist/runtime-dom.esm-bundler.js
@@ -1601,4 +1601,4 @@ const initDirectivesForSSR = () => {
   }
 } ;
 
-export { Transition, TransitionGroup, VueElement, createApp, createSSRApp, defineCustomElement, defineSSRCustomElement, hydrate, initDirectivesForSSR, render, useCssModule, useCssVars, vModelCheckbox, vModelDynamic, vModelRadio, vModelSelect, vModelText, vShow, withKeys, withModifiers };
+export { Transition, TransitionGroup, VueElement, createApp, createSSRApp, defineCustomElement, defineSSRCustomElement, hydrate, initDirectivesForSSR, render, useCssModule, useCssVars, vModelCheckbox, vModelDynamic, vModelRadio, vModelSelect, vModelText, vShow, withKeys, withModifiers, vShowOriginalDisplay, vShowHidden };
